src/memory/memory.c

```diff
@@ -335,6 +335,12 @@ bool memory_setup(const memory_interface_functions_t* ifs)
     chunk_0_t chunk = {0};
     CLEANUP_CHUNK(chunk);
     _read_chunk(CHUNK_0_PERMANENT, chunk_bytes);
+    // Factory setup image always does a factory reset of HWW
+#if FACTORYSETUP == 1
+    if (!memory_reset_hww()) {
```

Comment:
this is not a factory reset. `reset_reset()` is :thinking: not sure I like this change, it shouldn't be necessary to run it.

Reply:
Can I do a factory reset with the factory setup image?

Reply:
Or should we have a separate factory reset image?

Reply:
I removed this code, but I really think a device should be factory reset after running the factory setup. Perhaps there is some more appropriate way to achieve that.

Reply:
@benma would you like to resolve this?